app/realm/utils/post.js

```diff
@@ -46,7 +46,7 @@ export function postDataToRealm(post, owner) {
 
     if (post.metadata?.reactions) {
         post.metadata.reactions.forEach((r) => {
-            const id = `${post.id}-${owner.id}-${r.emoji_name}`;
+            const id = `${post.id}-${owner?.id}-${r.emoji_name}`;
```

Comment:
Why this change? This will create a string with `undefined`, definitely looks wrong

Reply:
The app would randomly crash due to owner being undefined at this point. It would happen randomly.